serverless/pages/what-is-elasticsearch-serverless.asciidoc

```diff
@@ -1,32 +1,61 @@
-////
-To be rewritten/refined
-////
+// ℹ️ THIS CONTENT IS RENDERERED ON THE index-serverless-elasticsearch.asciidoc PAGE
+// Use the id <<what-is-elasticsearch-serverless>> to link to this page
 
 // :description: Build search solutions and applications with {es-serverless}.
 // :keywords: serverless, elasticsearch, overview
 
 preview:[]
 
-{es} allows you to build custom applications. Whether you have structured or unstructured text, numerical data, or geospatial data, {es} can efficiently store and index it in a way that supports fast searches.
-
-.Understanding {es-serverless}
-[IMPORTANT]
+[TIP]
 ====
-Refer to <<elasticsearch-differences,Serverless differences>> and <<elasticsearch-technical-preview-limitations,Technical preview limitations>> for important details, including features and limitations specific to {es-serverless}.
+If you haven't used {es} before, you might want to learn the basics in the https://www.elastic.co/guide/en/elasticsearch/reference/current/elasticsearch-intro.html[main {es} documentation].
```

Comment:
I'd move this note below the intro paragraphs. I'd also simplify it: 

```suggestion
To learn the basics of {es}, refer to the https://www.elastic.co/guide/en/elasticsearch/reference/current/elasticsearch-intro.html[core {es} documentation].
```

Reply:
++ to brevity

I might reword again. LMKWYT.

In terms of placement, I kinda want it to be an immediate escape hatch for users that don't know what Elasticsearch is already.

Reply:
imo they don't know that they're confused yet if this note is first 🤷 

edit works for me

Reply:
"they don't know that they're confused yet" 😆 

I want to redirect them in their pre-confusion state. I'll have another look with fresh eyes tomorrow re placement.